src/software/world/refbox_constants_test.cpp

```diff
@@ -11,12 +11,12 @@ TEST(RefboxConstants, test_get_name_of_refbox_constants)
             RefboxGameState state = static_cast<RefboxGameState>(i);
             name(state);
         }
-        catch (std::invalid_argument)
+        catch (std::invalid_argument &)
```

Comment:
I'm fine with this, just curious why it was needed for Arch support? Or was it totally unrelated?

Reply:
GCC 9 warns on catching polymorphic exceptions by value.